### ISSUE TITLE
Styling issue of teasers placed in the highlight section

### ISSUE THREAD

**Issue by thomaslarsson** [sulu/sulu#3420]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 1.5.4
| Browser Version | Chrome 58.0.3029.110

#### Actual Behavior

The styling of teasers and media selections have issues in the admin panel if placed in the highlight section. 

[Screenshot of the admin panel](http://imgur.com/a/rO3Ru).

#### Expected Behavior

For the highlight section not to impact content type admin design.

#### Steps to Reproduce

```
<?xml version="1.0" ?>
<template xmlns="http://schemas.sulu.io/template/template"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">

    <key>homepage</key>

    <view>templates/homepage</view>
    <controller>SuluWebsiteBundle:Default:index</controller>
    <cacheLifetime>2400</cacheLifetime>

    <meta>
        <title lang="en">Homepage</title>
        <title lang="de">Startseite</title>
    </meta>

    <properties>
        <section name="highlight">
            <properties>
                <property name="title" type="text_line" mandatory="true">
                    <meta>
                        <title lang="en">Title</title>
                        <title lang="de">Titel</title>
                    </meta>
                    <params>
                        <param name="headline" value="true"/>
                    </params>

                    <tag name="sulu.rlp.part"/>
                </property>

                <property name="teasers" type="teaser_selection">
                    <meta>
                        <title lang="en">Teasers</title>
                    </meta>
                </property>

                <property name="link" type="single_internal_link">
                    <meta>
                        <title lang="en">Link</title>
                    </meta>
                </property>

                <property name="url" type="resource_locator" mandatory="true">
                    <meta>
                        <title lang="en">Resourcelocator</title>
                        <title lang="de">Adresse</title>
                    </meta>

                    <tag name="sulu.rlp"/>
                </property>
            </properties>
        </section>

        <property name="article" type="text_editor">
            <meta>
                <title lang="en">Article</title>
                <title lang="de">Artikel</title>
            </meta>
        </property>
    </properties>
</template>
```

#### Possible Solutions

Partial solution: Removing the highlight section currently removes the issue.

**Comment by wachterjohannes** [comment]
it looks like that this not only happens for teaser - it happens also for all the selections (article, contact, internal-links ...)

**Comment by wachterjohannes** [comment]
closed in favor of #3456
